slices_demo1/README.md

```diff
@@ -1,26 +1,38 @@
 # Demo1 README
 
-## 前言：
+## Preface 前言
+
+This article is based on the principle of Github open source P4Demo: Source_Routing.
+
+Before proceeding to the next step, please read the production of the Github project : [Github: SourceRouting](https://github.com/p4lang/tutorials/tree/master/SIGCOMM_2015#exercise-1-source-routing)
```

Comment:
It will be better if you use the word "introduction" instead of the "production".